JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -1,18 +1,92 @@
+    @IBOutlet private var kiwiQuantityLabel: KiwiQuantityLabel!
+    @IBOutlet private var mangoQuantityLabel: MangoQuantityLabel!
+    
+    // MARK: - LifeCycle
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(displayFruitQuantity),
+                                               name: Notification.Name("stcokDidChanged"),
+                                               object: nil)
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        displayFruitQuantity()
+    }
+    
+    // MARK: - Methods
+    
+    @objc private func displayFruitQuantity() {
+        let fruitQuantityLabels: [FruitQuantityLabel] = [strawberryQuantityLabel,
+                                                         bananaQuantityLabel,
+                                                         pineappleQuantityLabel,
+                                                         kiwiQuantityLabel,
+                                                         mangoQuantityLabel]
+        
+        for label in fruitQuantityLabels {
+            guard let furit = label.fruit() else { continue }
+            let fruitQuantity = juiceMaker.fruitStore.quantity(of: furit)
+            label.text = String(fruitQuantity)
+        }
+    }
+    
+    @IBAction func pressOrderButton(_ button: JuiceButton) {
+        guard let menu = button.pickMenu() else { return }
+        do {
+            try juiceMaker.makeJuice(menu: menu)
+            displayAcceptAlert(menuName: menu.description)
+        } catch {
+            displayFillStockAlert()
+        }
+    }
+    
+    private func displayAcceptAlert(menuName: String) {
+        let alert = UIAlertController(
+            title: "\(menuName) 쥬스 나왔습니다",
+            message: "맛있게 드세요!",
+            preferredStyle: UIAlertController.Style.alert
+        )
+        alert.addAction(UIAlertAction(title: "감사합니다", style: .default))
+        present(alert, animated: true, completion: nil)
+    }
+    
+    private func displayFillStockAlert() {
+        let alert = UIAlertController(
+            title: "재료가 부족해요",
+            message: "재고를 수정할까요?",
+            preferredStyle: .alert
+        )
+        alert.addAction(UIAlertAction(title: "예", style: .default, handler: { _ in
+            self.presentStockManager()
+        }))
+        alert.addAction(UIAlertAction(title: "아니요", style: .destructive))
+        present(alert, animated: true, completion: nil)
+    }
+    
+    private func presentStockManager() {
+        guard let presentViewController = self.storyboard?.instantiateViewController(identifier: "StockManagement") else { return }
+        self.present(presentViewController, animated: true, completion: nil)
     }
```

Comment:
```suggestion
    private func presentStockManagerViewController() {
        guard let stockManagerViewController = self.storyboard?.instantiateViewController(identifier: "StockManagement") else { return }
        self.present(presentViewController, animated: true, completion: nil)
    }
```
화면을 띄울땐 그냥 해당 화면 이름 그대로 메서드로 이름짓는게 훨씬 이해하기 빠릅니다. 
지금은 모델을 가지고 있는 StockManager 를 present 하는 것 같네요!

Reply:
수정하겠습니다! 🙂